Code/Tools/ProjectManager/Source/GemCatalog/GemFilterWidget.cpp

```diff
@@ -487,7 +498,7 @@ namespace O3DE::ProjectManager
             const QString& feature = elementNames[i];
             QAbstractButton* button = buttons[i];
 
-            // Adjust the proxy model and enable or disable the clicked feature used for filtering.
```

Comment:
Fyi, we are looking to specifically avoid the term 'disabled' so disable would not fall under that.